rust/Anchor.toml

```diff
@@ -27,8 +30,7 @@ staking = "stk9HFnKhZN2PZjnn5C4wTzmeiAEgsDkbqnHkNjX1Z4"
 test = "yarn run ts-mocha -p ./tsconfig.json -t 1000000 tests/**/*.ts"
 
 [workspace]
-members = ["item", "namespace", "matches", "staking"]
-types = "../js/lib/src/idls/"
+members = ["item", "player", "namespace", "matches", "staking"]
```

Comment:
I have been leaving out player until J is done with his big changes, I suggest we do not add it until then

Reply:
They won't crash with player program I think.

Reply:
It doesn't make any program changes actually, but staking program needs player program, so I had to build player program to test. :)